### PR TITLE
Marketing optimization

### PR DESCRIPTION
**Pull request by plondon**
Any files with a label TEMP, or code with comments surrounded with TEMP will be removed in the next iteration of the marketing page. It looks like a lot of lines of code have been added but most of them are from SVG files.

todo

- [x] add tests

### CODE REVIEW

File: assets/js/directives/scroll-in-view.directive.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ function scrollInView($window, $timeout) {
   const directive = {
     restrict: 'E',
     templateUrl: 'templates/scroll-in-view.jade',
+    replace: true,
```
Comment (jtormey) [inline assets/js/directives/scroll-in-view.directive.js]: Why does this need to replace?

**Comment by jtormey** [comment]
ACK

File: assets/js/directives/scroll-in-view.directive.js (hunk 1 above)
Comment (plondon) [inline assets/js/directives/scroll-in-view.directive.js]: It seemed like have <scroll-in-view><div class="scroll-in-view"> was a unnecessary.

File: app/index.jade [hunk 2]
```diff
@@ -176,9 +179,27 @@ head
...
   <!-- @endif -->
+
+body(ng-controller="AppCtrl", ui-view="body")
   
+  //- This does not depend on Angular and will show as soon as the images are done loading.
+  <!-- @if PRODUCTION !>
+  .blocket-loading
+    div
+      img.full(src="/img/blocket-full-4847e61888d2d3c9f33f14840f92783d1c91dea2.png")
+      img(src="/img/blocket-no-boosters-c3f2a027ddfdbbce8eba8fc5da84c65d89d14fbc.png")
```
Comment (Sjors) [inline app/index.jade]: No need to disable this in non-production. Also no need to put the sha hash in the image url's. This is handled by `grunt dist_unsafe` / `grunt release`.

File: app/index.jade [hunk 3]
```diff
@@ -176,9 +179,27 @@ head
...
+  <!-- @endif -->  
+
+  noscript
+    img(src="/img/logo-bw.png")
+    h3 Please enable JavaScript to use the Blockchain Wallet
+
+  //- Load application.min.js in the body async.
+  <!-- @if PRODUCTION !>
+  script(src='js/application.min.js')
```
Comment (Sjors) [inline app/index.jade]: If you want to load javascript in the body on production, you should do the same in dev. I'll add a commit.

Comment (Sjors) [inline app/index.jade]: Turns out that there are side-effects when loading Javascript in the body. For instance `$rootScope.rootURL =` in `wallet.service.js` gets called before ng-init has set the value.

cc @jtormey

Comment (jtormey) [inline app/index.jade]: Interesting, good to know. I'll see how we can deal with that.

**Pull request merged**